jsx/connectors/jack.jsx

```diff
@@ -151,6 +154,6 @@ Jack.propTypes = {
   name: PropTypes.string.isRequired,
   pins: PropTypes.array.isRequired,
   jack_id: PropTypes.string.isRequired,
-  remove: PropTypes.func.isRequired,
+  remove: PropTypes.func,
```

Comment:
I'd rather not remove app code to facilitate tests.  Try wrapping jest.fn() with a function for test coverage. 
See https://github.com/facebook/jest/issues/6329

Reply:
was a forget. Fixed the problem and found whyt this was not working.